### DISCUSSION TITLE
consensus/raft

### DISCUSSION THREAD

**Discussion by giscus** [isno/theByteBook#140] [General]
# consensus/raft

构建大规模高可用的分布式系统

https://www.thebyte.com.cn/consensus/raft.html

**Comment by dentonliu** [comment]
R{eliable|plicated|dundant}  应该把e放到外面 Re{liable|plicated|dundant}

**Reply by isno** [reply]
谢谢，已修正。

**Comment by chainray01** [comment]
6.2/6.3/6.4目录结构错误

**Reply by isno** [reply]
谢谢，已修正
